### DISCUSSION TITLE
We're TRACKING projects and FOLLOWING people, not VIEWING Twitter or SWIPING Tiktok

### DISCUSSION THREAD

**Discussion by SpriteOvO** [community/community#66301] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I just suddenly realized today that I should start thinking about a fallback option alternative to GitHub, maybe GitLab, maybe self-host.

[Achievements](https://github.blog/2022-06-09-introducing-achievements-recognizing-the-many-stages-of-a-developers-coding-journey/), [SocialMedia-like Feed](https://github.blog/changelog/2023-09-06-updates-to-your-github-feed/). Useless features are adding up and useful features are fading away.

I'm interested to see what the last straw will be, or GitHub could realize that people are WORKING, are CONTRIBUTING, not PLAYING. 😇

**Comment by port19x** [comment]
If you're gonna self-host, check out gitea. 
Trivial to host on docker and has an at least somewhat similar ui to github, github prior to this enshittification that is

**Comment by wangrongding** [comment]
# Github Old Feed  

💪 Using this script you can restore the old version of the feed.
https://github.com/wangrongding/github-old-feed  

<table>
  <tr>
      <td width="50%" align="center"><b>Before</b></td>
      <td width="50%" align="center"><b>After use</b></td>
  </tr>
  <tr>
     <td><img src="https://assets.fedtop.com/picbed/1694066442147.jpg"/></td>
     <td><img src="https://assets.fedtop.com/picbed/1694066437621.jpg"/></td>
  </tr>
</table>

**Comment by pouriap** [comment]
This thing on a platform such as GitHub is just pure insult. It's like bringing a clown to an office. It's like playing Nicki Minaj in a corporate meeting. This has nothing to do with this platform. People are doing serious work in here, putting in their precious free time to contribute to projects that benefit millions worldwide. And then you start this clown show as if this is all a joke to you.

**Reply by janbrasna** [reply]
… after a year+ of going through "product feedback" learning from people they don't want this, flipping the switch back all the time again and again… but the money was spent, so it's must have to launch even if all customers go elsewhere… 🤷
